src/dmd/frontend.h

```diff
@@ -1553,27 +1553,53 @@ class StoppableVisitor : public Visitor
 
 struct TargetC
 {
+    enum class Runtime : uint8_t
+    {
+        Unspecified = 0u,
+        Bionic = 1u,
+        DigitalMars = 2u,
+        Glibc = 3u,
+        Microsoft = 4u,
+        Musl = 5u,
+        Newlib = 6u,
+        UClibc = 7u,
+        WASI = 8u,
+    };
+
     uint32_t longsize;
     uint32_t long_doublesize;
     Type* twchar_t;
+    Runtime runtime;
     TargetC() :
         longsize(),
         long_doublesize(),
         twchar_t()
     {
     }
-    TargetC(uint32_t longsize, uint32_t long_doublesize = 0u, Type* twchar_t = nullptr) :
+    TargetC(uint32_t longsize, uint32_t long_doublesize = 0u, Type* twchar_t = nullptr, Runtime runtime = (Runtime)0u) :
```

Comment:
```suggestion
    TargetC(uint32_t longsize, uint32_t long_doublesize = 0u, Type* twchar_t = nullptr, Runtime runtime = Runtime.Unspecified) :
```

Reply:
No can do. `frontend.h` is autogenerated. cc @MoonlightSentinel  this seems to a limitation of `dtoh`

Reply:
> cc @MoonlightSentinel this seems to a limitation of dtoh

IIRC this is caused by an inconsistent rewrite of enum initializers. Will have a look again soon.

Reply:
Is the original kept around for reference?